opentelemetry/src/global/metrics.rs

```diff
@@ -44,26 +44,26 @@ pub fn meter(name: &'static str) -> Meter {
 /// - version specifies the version of the instrumentation scope if the scope has a version
 /// - schema url specifies the Schema URL that should be recorded in the emitted telemetry.
 ///
-/// This is a convenient way of `global::meter_provider().versioned_meter(...)`
+/// This is a convenient way of `global::meter_provider().library_meter(...)`
 ///
 /// # Example
 ///
 /// ```
-/// use opentelemetry::global::meter_with_version;
+/// use std::sync::Arc;
+/// use opentelemetry::global::library_meter;
+/// use opentelemetry::InstrumentationLibrary;
 /// use opentelemetry::KeyValue;
 ///
-/// let meter = meter_with_version(
-///     "io.opentelemetry",
-///     Some("0.17"),
-///     Some("https://opentelemetry.io/schemas/1.2.0"),
-///     Some(vec![KeyValue::new("key", "value")]),
+/// let library = Arc::new(
+///     InstrumentationLibrary::builder("io.opentelemetry")
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#get-a-meter

I think we should rename InstrumentationLibrary to InstrumentationScope everywhere, as Scope is what user provides.

Reply:
@lalitb @utpilla Please share your thoughts on this.

Reply:
Yes. `InstrumentationScope` also matches the term used in [proto file](https://github.com/open-telemetry/opentelemetry-proto/blob/7312bdf63218acf27fe96430b7231de37fd091f2/opentelemetry/proto/common/v1/common.proto#L69-L81).

Reply:
+1 for `InstrumentationScope`

Reply:
done